app/tests/TestCase/Controller/GatheringsControllerTest.php

```
@@ -155,6 +156,87 @@ public function testSavedGridViewFiltersGatheringsTodayOrAfter(): void
+                    ],
+                ],
+            ]),
+        ], $currentUser);
+        $this->assertNotFalse($gridView);
+        $this->assertSame(
+            ['schedule_window', 'gathering_type_id'],
+            array_column($gridView->getConfigArray()['filters'], 'field'),
+        );
+
+        $this->get('/gatherings/grid-data?' . http_build_query([
+            'view_id' => $gridView->id,
```

PHPStan (2.2.7)

[error] Cannot access property $id on App\Model\Entity\GridView|false.


(property.nonObject)